substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -530,68 +652,163 @@ use sp_std::convert::{TryFrom, TryInto};
 use tfchain_support::types::PublicIP;
 // Internal functions of the pallet
 impl<T: Config> Pallet<T> {
-    pub fn _create_node_contract(
-        account_id: T::AccountId,
+    pub fn _create_group(account_id: T::AccountId) -> DispatchResultWithPostInfo {
+        let twin_id = pallet_tfgrid::TwinIdByAccountID::<T>::get(&account_id)
+            .ok_or(Error::<T>::TwinNotExists)?;
+
+        let mut id = GroupID::<T>::get();
+        id = id + 1;
+
+        let new_group = types::Group {
+            id: id,
```

Comment:
with short notation you can write `id: id,` => `id,`

Reply:
Yes but it's more clear like this no? I had a compilation error that was complaining about a type not being right while the problem was actually that I did not add the property (in the declaration) that I added to the struct.

Reply:
Okay, then you can leave it :)